### PR TITLE
BD-3156 Landing pages beta

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3156**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+nav_title: Drag-and-drop Landing Pages
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:2]:
```suggestion
nav_title: Drag-and-Drop Landing Pages
```

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+> Using the drag-and-drop editor, you can create and customize a landing page to grow your audience and collect preferences directly in Braze.
+
+{% alert important %}
+Landing pages are currently in beta. Contact your Braze account manager if you're interested in participating in this beta.
+{% endalert %}
+
+![GIF demonstrating how components can be dragged and dropped into an editor to create a landing page.][1]{: style="max-width:70%;"}
+
+Check out our [Customer Ed Quick Tips]() video to get a jumpstart on creating Braze landing pages.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:20]:
```suggestion
Check out our [Customer Ed Quick Tips]() video to get a jumpstart on creating Braze landing pages.
```
Missing a link here! Also, curious about the styling of the text, is that a Braze(tm) thing? "Customer Education Quick Tips"?

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+#### Site details
+
+Set up metatags to customize how your page appears on the browser tab and optimize for search engine results. These will be visible to your customers.
+
+We suggest following these best practices:
+
+| Detail | Description | Recommendations |
+| --- | --- |
+| Site title | The title that displays on the brower tab. | Use a maximum of 60 characters. |
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:44]:
```suggestion
| Site title | The title that displays on the browser tab. | Use up to 60 characters. |
```

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+You can set styles to be applied across all relevent component blocks in your landing page from the **Page container** tab. These styles will be used everywhere on your page except where you override them with a specific block.
+
+We recommend setting up page container-level styles before you customize styles at the block level. You can also add a background image for the entire page.
+
+![The page container with options to customize background images, colors, border details, and content styling.][6]{: style="max-width:30%;"}
+
+### Step 4: Preview your landing page
+
+You can preview your landing page in the editor's **Preview** tab, but testing functionality is disabled for the beta. After you save your landing page as a draft, you can visit the URL under **URL handle** to preview the page on your browser. You can also share the URL with collaborators.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:84]:
```suggestion
You can preview your landing page in the editor's **Preview** tab, but testing functionality is disabled for the beta. After saving your landing page as a draft, you can visit the URL under **URL handle** to preview the page on your browser. You can also share the URL with collaborators.
```

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+If a user inputs an invalid form value (such as unaccepted special characters), they will see a generic error indicator that isn't customizable and won't be able to submit the form. You can view the error behavior in the **Preview** tab and the preview landing page.
+
+## Merging users created from your landing page
+
+During the beta, each form submission on a landing page will create a new anonymous user profile in Braze. If a user with the same email address already exists, you can merge the new user profile into the existing profile using the [`/users/merge`]({{site.baseurl}}/api/endpoints/user_data/post_users_merge#merging-unidentified-user) endpoint. To learn about the different ways to deduplicate users in Braze, see [Duplicate users]({{site.baseurl}}/user_guide/engagement_tools/segments/user_profiles/duplicate_users).
+
+## Considerations
+
+- The landing page body size can be a maximum of 1 MB.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:112]:
```suggestion
- The landing page body size can be up to 1 MB.
```

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+## Frequently asked questions
+
+### What happens when a user submits their information on the landing page?
+
+When a user submits a form, a new Braze user profile is created with the submitted user data.
+
+{% alert note %}
+During the beta, Braze will create a new user with each form submission regardless of where the user clicks on the landing page from.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:121]:
```suggestion
During the beta, Braze will create a new user with each form submission regardless of where the user clicks on the landing page.
```

**Review by lydia-xie — APPROVED** [review]
looks good! left some suggestions for your review

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md (hunk 2 above)
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:20]:
```suggestion
```

**Pull request merged**